validator/validator-main.protoascii

```diff
@@ -7054,6 +7056,14 @@ error_formats {
   code: GENERAL_DISALLOWED_TAG
   format: "The tag '%1' is disallowed except in specific forms."
 }
+error_formats {
+  code: LTS_RUNTIME_ONLY
+  format: "Extension '%1' must use LTS to match LTS runtime."
+}
+error_formats {
+  code: LTS_EXTENSION_ONLY
+  format: "Extension '%1' may not use LTS with non-LTS runtime."
```

Comment:
Best guess on error message content/format; feel free to suggest edits/alternatives.